### DISCUSSION TITLE
Posibility of getting information with pyFluent from geometry, and mesh

### DISCUSSION THREAD

**Discussion by wkozek** [ansys/pyfluent#3592] [Q&A] [answered]
Hello all, 

I am writing to ask for some help with the automatization process. 
 
I am currently working on mesh automatization, and I have some questions: 
1.            Is it possible to check NamedSelections assigned into geometry ( what is importing to Fluent Mesher)? If yes, how to do it? 
2.            How do you prepare a short report on mesh quality? How do you get some info about a number of elements and quality? For example, : skewness, aspect ratio or orthogonal quality?
 
In the first point, I would like to get a list that would indicate areas, e.g. where the mesh is densified using face_sizing.
2nd point would help me get quick statistics to check mesh parameters in case of background meshing.
 
I really appreciate any guidance or help.

**Comment by hpohekar** [comment, marked as answer]
@wkozek 

You can use the `meshing_utilities` API for above mentioned points.

Please refer https://github.com/ansys/pyfluent/blob/main/tests/test_meshing_utilities.py for more details.

cc: @mkundu1 @seanpearsonuk
